examples/step-18/step-18.cc

```diff
@@ -1333,6 +1288,8 @@ namespace Step18
           std::pair<double, std::string>(present_time, pvtu_master_filename));
         std::ofstream pvd_output("solution.pvd");
         DataOutBase::write_pvd_record(pvd_output, times_and_names);
+        std::ofstream visit_output("solution.visit");
+        DataOutBase::write_visit_record(visit_output, times_and_names);
```

Comment:
@tjhei Like this? In the other visit-cases this does not work. Here it is used as an equivalent for `.pvd`, there is is used as an equivalent for `.pvtu`.